lib/babel-plugin-inline-svg.js

```diff
@@ -48,9 +49,12 @@ module.exports = ({ types }) => ({
           ? maybeOptimized
           : namespaceIds(varName, maybeOptimized);
 
+        const final = inlineInImgSrc
+         ? dataURIImgSrc + Buffer.from(sauce).toString("base64")
+         : sauce;
```

Comment:
A few things here:

1. Instead of using string concatenation with `+`, lets use backticks like we have elsewhere in the repo
1. Although it's super simple code, I think it should be it's own standalone function inside `lib`, maybe `createDataURI.js`? It would then fit with the rest of the transformers and we could write standalone tests if we wanted
1. I recently changed how the transformer functions are applied to the output, to make them easier to reason about and to make it simpler to add new ones, so you'll need to update this code to match...

Putting all of the above together, I think something like the following would be better:

```js
// lib/createDataURI.js
module.exports = (input) => `data:image/svg+xml;base64,${Buffer.from(input).toString("base64")}`

// then, back in lib/babel-plugin-inline-svg.js using the new compose functionality
...
const output = compose(
  dataURI ? createDataURI : noop,
  disableNamespaceIds ? noop : curry(namespaceIds)(varName),
  disableSVGO ? noop : curry(optimize)(state.opts.svgo)
)(input);
```

Makes sense?

Reply:
I'll make the change, but I have done this way following the original code. Maybe this is a recent change that wasn't available... It's much better, though!